common/cosmwasm-smart-contracts/mixnet-contract/src/interval.rs

```diff
@@ -60,14 +60,22 @@ pub(crate) mod string_rfc3339_offset_date_time {
     }
 }
 
+#[cfg_attr(feature = "generate-ts", derive(ts_rs::TS))]
+#[cfg_attr(
+    feature = "generate-ts",
+    ts(export_to = "ts-packages/types/src/types/rust/Interval.ts")
+)]
 #[derive(Clone, Copy, Debug, Deserialize, PartialEq, Eq, Serialize)]
 pub struct Interval {
     id: IntervalId,
     epochs_in_interval: u32,
 
+    // TODO add a better TS type generation
+    #[cfg_attr(feature = "generate-ts", ts(type = "undefined"))]
```

Comment:
is `undefined` the correct way of handling it? if we don't care about that field, how about explicitly ignoring it? (or maybe doing some string conversion?)